datasets/coco2014/metadata.json

```diff
@@ -223,6 +226,83 @@
           "references": "#{split_enums}"
         }
       ]
+    },
+    {
+      "name": "categories",
+      "@type": "ml:RecordSet",
+      "source": "#{instancesperson_keypoints_annotations/categories}",
+      "key": "#{id}",
+      "field": [
+        {
+          "name": "id",
+          "@type": "ml:Field",
+          "description": "The ID of the category",
+          "dataType": "sc:Integer",
+          "source": "#{id}"
+        },
+        {
+          "name": "name",
+          "@type": "ml:Field",
+          "description": "The name of the category.",
+          "dataType": [
+            "sc:Text",
+            "sc:name"
+          ],
+          "source": "#{name}"
+        },
+        {
+          "name": "supercategory",
+          "@type": "ml:Field",
+          "description": "The name of the supercategory.",
+          "dataType": [
+            "sc:Text",
+            "sc:name"
+          ],
+          "source": "#{supercategory}"
+        }
+      ]
+    },
+    {
+      "name": "annotations",
+      "@type": "ml:RecordSet",
+      "source": "#{instancesperson_keypoints_annotations/annotations}",
+      "key": "#{id}",
+      "field": [
+        {
+          "name": "id",
+          "@type": "ml:Field",
+          "description": "The ID of the annotation.",
+          "dataType": "sc:Integer",
+          "source": "#{id}"
+        },
+        {
+          "name": "category_id",
+          "@type": "ml:Field",
+          "description": "The ID of the category.",
+          "dataType": "sc:Integer",
+          "source": "#{category_id}",
+          "references": "#{categories}"
+        },
+        {
+          "name": "image_id",
+          "@type": "ml:Field",
+          "description": "The ID of the image.",
+          "dataType": "sc:Integer",
+          "source": "#{image_id}",
+          "references": "#{images}"
+        },
+        {
+          "name": "bbox",
+          "@type": "ml:Field",
+          "description": "The bounding box around annotated object[s].",
+          "dataType": "ml:BoundingBox",
+          "source": {
+            "data": "#{bbox}",
+            "format": "[x,y,width,height]"
```

Comment:
I like Keras's bounding boxes format: https://keras.io/api/keras_cv/bounding_box/formats/ `CENTER_XYWH `, `XYWH`, etc.

Reply:
Excellent, let's go for that!